### ISSUE TITLE
Cold water exposure, repeated

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10142]
**Preferred term label:**
Cold water exposure, repeated

**Synonyms**
Surfing, diving, kayaking, swimmers

**Definition (free text, please give PubMed ID)**
There are some UMLS codes for cold water but they are as a treatment, not cause of illness.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Surfer's ear

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000041
- label: Repeated old water exposure,
- parents: [Exposure history (HP:4000110)](https://hpo.jax.org/app/browse/term/HP:4000110)
- pmid: 
- def: A history of repeated exposure to cold water, as may result, for instance, from surfing, diving, kayaking, or swimming.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10142
- synonyms:

**Comment by MickeySegal** [comment]
Typo.  Should be label: Repeated **c**old water exposure
